src/calculateRentalCost.js

```diff
@@ -4,7 +4,19 @@
  * @return {number}
  */
 function calculateRentalCost(days) {
-  // write code here
+  const cost = 40;
+  const LONG_TERM = 7;
+  const LONG_TERM_DISCOUNT = 50;
+  const SHORT_TERM = 3;
+  const SHORT_TERM_DISCOUNT = 20;
+
```

Comment:
Create a variable for base price and use it everywhere
```suggestion
  const SHORT_TERM_DISCOUNT = 20;
  const basePrice = days * cost;
  
```